### ISSUE TITLE
New preset: shop=hvac

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/iD#8071]
https://wiki.openstreetmap.org/wiki/Tag:shop%3Dhvac

**Comment by quincylvania** [comment]
@maro-21 Hmm 144 really isn't that many uses. Is this in the NSI or something?

**Comment by maro-21** [comment]
Just a missing shop tag. The tag isn't popular because the shop isn't that popular either ;) But such shops exist.

**Comment by quincylvania** [comment]
@maro-21 Yes, there are thousands of different types of shops, but there's a maintenance cost to every preset so our goal isn't really to support all of them…
